docs/assets/js/main.js

```diff
@@ -75,9 +75,45 @@ function injectFooterCopywrite() {
     footer.innerHTML = `&copy;2008-${new Date().getFullYear()} Expensify, Inc.`;
 }
 
+// Count property of --scroll-y variable
+window.addEventListener('scroll', () => {
+    document.documentElement.style.setProperty('--scroll-y', `${window.scrollY}px`);
+});
+
+function openSidebar() {
+    document.getElementById('sidebar-layer').style.display = 'block';
+    document.getElementById('gsc-i-id1').placeholder = 'Search for something...';
+    // Make body unscrollable
+    const scrollY = document.documentElement.style.getPropertyValue('--scroll-y');
+    const body = document.body;
+    body.style.position = 'fixed';
+    body.style.top = `-${scrollY}`;
+}
+
+function closeSidebar() {
+    document.getElementById('sidebar-layer').style.display = 'none';
+    // Make body scrollable again
+    const body = document.body;
+    const scrollY = body.style.top;
+    body.style.position = '';
+    body.style.top = '';
+    window.scrollTo(0, parseInt(scrollY || '0', 10) * -1);
```

Comment:
Add a little comment about why we're scrolling to the "Y" value in `scrollTo` and what represents that value